Lib/glyphsLib/builder/instances.py

```diff
@@ -71,6 +71,13 @@ def _to_designspace_instance(self, instance):
                 fname = self.instance_dir + "/" + fname
             ufo_instance.filename = fname
 
+    # Since Glyphs Format v3 the "postscriptFontName" property
+    # is stored in "properties" instead of "customParameters".
+    for p in instance.properties:
+        key, value = p.key, p.value
+        if key == "postscriptFontName":
+            ufo_instance.postScriptFontName = value
```

Comment:
Glyphs Python API has [`GSInstance.fontName`](https://docu.glyphsapp.com/#GSInstance.fontName), which is probably what should have been used here (see also the comment above), but I think this OK for now.